data/marameref.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "Amarachi Omereife",
+  "bio": "Seasoned Software Engineer and Azure Cloud Engineer **Proud Techie**. Health & Wellness Enthusiast",
+  "socials": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/marameref",
+      "icon": "marameref"
+    },
+    {
+      "name": "Follow me on linkedin",
+      "url": "https://www.linkedin.com/in/amarachi-crystal-omereife-416a081a1/",
+      "icon": "Amarachi Crystal Omereife"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://instagram.com/marameref",
+      "icon": "marameref"
+    } 
+  ],
+
+  "Links": [
```

```suggestion
  "links": [
```
It should be links instead of Links